demo/src/search/search-bar-component.js

```diff
@@ -0,0 +1,90 @@
+import { html, LitElement, unsafeCSS } from 'lit';
+import { animations, theme } from '../theme/theme';
+import componentCSS from 'bundle-text:./search-bar-component.scss';
+import Pillarbox from '../../../src/pillarbox';
+
+export class SearchBarComponent extends LitElement {
+  static properties = {
+    bu: { type: String },
+    query: { type: String }
+  };
+
+  static styles = [theme, animations, unsafeCSS(componentCSS)];
+
+  constructor() {
+    super();
+    this.bu = 'rsi';
```

Comment:
Can't we add a constant for the default bu as it is used more than once in the code?

Reply:
Done.